cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -829,6 +830,14 @@ def _command_runner(
                 rendered_command_code,
                 **kwargs,
             )
+        elif command.action == "plan":
+            return self.with_context(
+                disable_recursive_plan=True
```

Comment:
Better to rename it into something more relevant. Eg `prevent_plan_recursion`.
Otherwise it may look like we can disable or enable recursion which is not true.